### PR TITLE
Fixed lists alignment error

### LINKED ISSUE #3264: Bad UI of /home/lists page
In addition to #3070 
As shown in the screenshot the /home/lists page has many spacing alignment errors. The page should be able to project a clean list of various *LISTS* created by users.

### Evidence / Screenshot (if possible)
lists page

lists page 2

### Relevant url?

https://openlibrary.org/lists
### Steps to Reproduce

1. Go to open library home page
2. scroll down to *About the Library*
3. Click on the graph showing *Lists created*

* Actual: a page showing some recommendations of lists pops up with a lot of bad UI.
* Expected: expecting to see some sort of aligned way to show some already existing lists.

### Details

- **Logged in (Y/N)?** Y
- **Browser type/version?** Chrome
- **Operating system?** Windows
- **Environment (prod/dev/local)?** prod

### Proposal & Constraints
1. A new page with some good spacing alignment should be made with a recommendation list of *Lists*. 
2. Some customization in the design to well define the lists n the recent activity related to the lists at the bottom of the page, as highlighted in the second screenshot above.

### Stakeholders
@mekarpeles

### PR DESCRIPTION
**Pull request by Yashs911**
Closes #3264 

fixed bad UI
### Screenshot

ol sol lists

### CODE REVIEW

File: static/css/page-user.less [hunk 1]
```diff
@@ -104,6 +104,16 @@ div#subjectsPage{
   }
 }
 
+div#lists{
+  columns: 1;
+  @media only screen and (min-width: @width-breakpoint-tablet) {
+    columns: 2;
+  }
+  @media only screen and (min-width: @width-breakpoint-desktop) {
```
Comment (jdlrobson) [inline static/css/page-user.less:112]: This appears to render the whole list in a single column  for me at desktop breakpoint.

Screen Shot 2021-01-07 at 5 45 20 PM

**Review by jdlrobson — CHANGES_REQUESTED** [review]
This doesn't seem to fix the issue for me..

File: static/css/page-user.less [hunk 2]
```diff
@@ -104,6 +104,16 @@ div#subjectsPage{
   }
 }
 
+div#lists{
+  columns: 1;
```
Comment (jdlrobson) [inline static/css/page-user.less:108]: If you use columns you shouldn't use the float:left above.

Also you'll probably need a fixed height.

Have you considered a flex box solution for this? That's probably going to be the best result here.

Comment (Yashs911) [reply static/css/page-user.less]: @jdlrobson I have made some changes.

**Comment by Yashs911** [comment]
@jdlrobson I was wondering why do we call books **seeds**?? I mean for users it should be **books**.
and I think we should style the lists page something like this
image

I mean most of the description is empty so why not remove it.

**Comment by jdlrobson** [comment]
Looks like this should do the trick.

how did you test this locally? Active lists is empty for me and creating lists doesn't seem to have populated it.

I manually applied to the production .lists URL for the time being.

Can we add a media query to make these full screen on mobile resolutions?:

Screen Shot 2021-01-08 at 9 14 19 AM

**Review by jdlrobson — APPROVED** [review]

**Comment by Yashs911** [comment]
@jdlrobson I manually applied all changes using the chrome developer tool.

**Comment by Yashs911** [comment]
@jdlrobson will this work 
image

**Comment by jdlrobson** [comment]
I see it looks a bit odd centered, huh?

I think it would be better to apply width: 100%. and make it 280px when `@media only screen and (min-width: @width-breakpoint-tablet) {`

The `.listEntry .data` rule should also be 100%, so that the width is determined by .listEntry not one of its children.

**Comment by Yashs911** [comment]
@jdlrobson 
for mobile:
image
for tablet it will look something like this:
image
for desktop:
image

File: static/css/components/list-entry.less [hunk 3]
```diff
@@ -3,9 +3,11 @@
  * https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library#listentry
  */
 .listEntry {
-  width: 280px;
-  margin: 0 30px 35px 0;
-  float: left;
+  width: 100%;
+  @media only screen and (min-width: @width-breakpoint-tablet) {
```
Comment (jdlrobson) [inline static/css/components/list-entry.less:7]: Please don't nest this. It makes it hard to find.

Style-listically, please put these at the end of the file.

File: static/css/components/list-entry.less [hunk 4]
```diff
@@ -3,9 +3,11 @@
...
+    width: 280px;
+  }
+  margin: 20px 10px;
```
Comment (jdlrobson) [inline static/css/components/list-entry.less:10]: Always put a space after selectors (not sure why stylelint doesn't complain about that)

**Review by jdlrobson — CHANGES_REQUESTED** [review]
A few style  nitpicks.

**Comment by Yashs911** [comment]
@jdlrobson Done.

**Review by jdlrobson — APPROVED** [review]
need to verify locally - and need to work out how to populate lists before doing that.

**Comment by Yashs911** [comment]
@jdlrobson I made 6 lists locally.

**Comment by Yashs911** [comment]
@jdlrobson any update?

**Comment by jdlrobson** [comment]
I finally managed to get setup with ability to test this locally and verify. 

Development environment for this feature is in a sorry state (help fixing that would be greatly appreciated as it would make reviewing future patches easier).

Anyway this looks good! For future reference, In case it's helpful this is how I got to test this locally:

Replace following line in openlibrary/templates/lists/home.html
```
<a href="$list.key" class="title results">$:_('%(listname)s </a> from <a href="%(key)s">%(owner)s</a>', key=owner.key, listname=list.name, owner=owner.displayname)
```
with
```
                    <a href="$list.key" class="title results">$:_('%(listname)s </a> from <a href="#">%(owner)s</a>', listname=list.name, owner="user")
```

Then in docker exec web bash
```
./scripts/copydocs.py "/type/lists" 
# repeat for about 8 lists I find on http://openlibrary.org/lists
./scripts/copydocs.py "/people/kimberly888/lists/OL191786L"
```

Then restarted the docker memcached service.

Ran JS code to correct image thumbnails:
```
Array.from(document.querySelectorAll('.cover img')).forEach((a) => a.setAttribute('src',a.getAttribute('src').replace('http://covers:7075/', 'https://covers.openlibrary.org/')))
```

**Pull request merged**